openkfw/TruBudget
# Add security restrictions for passwords

# Changes
If a user changes his/her password, the new password has to adhere to the following conditions: 
- Minimal length of 8 characters
- At least one letter
- At least one number

Closes #370

# How to test
Change a user's password - you should not be able to create one that does not adhere to the above rules.
